spec/spec_helper.rb

```diff
@@ -57,21 +59,7 @@
   config.include AwesomeSpawn::SpecHelper, :uses_awesome_spawn => true
   config.before(:each, :uses_awesome_spawn) { disable_spawning }
 
-  # factory methods
-
-  def make_workflow(ctx, payload)
-    Floe::Workflow.new(make_payload(payload), ctx)
-  end
-
-  def make_payload(states)
-    start_at ||= states.keys.first
-
-    {
-      "Comment" => "Sample",
-      "StartAt" => start_at,
-      "States"  => states
-    }
-  end
+  config.include FloeFactory
```

Comment:
Just naming, but what about `FloeHelper` instead?  Especially in specs Factory makes me think of FactoryBot which this isn't and AwesomeSpawn have `AwesomeSpawn::SpecHelper` above

Reply:
Or `Floe::SpecHelper`?

Reply:
Yeah wasn't sure if he was trying to keep it split out into specific concerns, maybe `Floe::SpecHelper` if you want all in one or `Floe::WorkflowHelper` if you want to keep things split out

Reply:
`AwesomeSpawn::SpecHelper` is used by non-AwesomeSpawn code to help better leverage the `AwesomeSpawn` library.

So if we provided helpers to be used by the workflow or other non-floe code, then that could be a great option.

So I'm still not sure what to call this.

Reply:
@kbrock That's a good point we should reserve Floe::SpecHelper for a public module we might want to share for callers to test floe stuff in the same way we have AwesomeSpawn::SpecHelper

I checked our other gems, and everywhere else, we either just call it by feature name followed by helper (not scoped) (e.g. `DatabaseHelper`, `ConnectionHelper`), or we have it in a Spec::Support namespect (e.g. `Spec::Support::DatabaseHelper`). Helper as a term is pretty consistent.

So I'm thinking spec/support/workflow_helper.rb and inside either WorkflowHelper or Spec::Support::WorkflowHelper.  Honestly though, I'm fine with anything

Reply:
I tried out `CommonMethods` but I do like the `Helpeb word suggestion.
I thought we didn't do namespacing.